src/lm_buddy/jobs/evaluation/prometheus.py

```diff
@@ -84,18 +88,21 @@ def instruction_to_prompt(config: PrometheusJobConfig, instruction: str) -> str:
 
 
 def get_response_with_retries(
-    config: PrometheusJobConfig, client: OpenAI, prompt: str, max_retries: int
+    config: PrometheusJobConfig,
+    client: OpenAI,
+    engine: str,
+    prompt: str,
```

Comment:
Thanks for catching the redundant max_retries! It was a byproduct of tests I did before putting it in the config and then I missed it when refactoring 🙏